packages/language/src/syntax-tree/ast.ts

```diff
@@ -1308,12 +1273,14 @@ export function createDimensions(): Dimensions {
 export interface DimensionsDataAttribute extends AstNode {
   kind: SyntaxKind.DimensionsDataAttribute;
   dimensions: Dimensions | null;
+  dimensionsToken: Token | null;
 }
 export function createDimensionsDataAttribute(): DimensionsDataAttribute {
   return {
     kind: SyntaxKind.DimensionsDataAttribute,
     container: null,
     dimensions: null,
+    dimensionsToken: null,
```

Comment:
Question: Is there a specific reason to store the token on the attribute itself? Otherwise, I would recommend to store it on the `Dimensions` ast node, which would also make the parsing less complicated.